examples/kubernetes/snapshot/README.md

```diff
@@ -0,0 +1,100 @@
+# Snapshots
+
+## Context
+
+Volume snapshots allow users to quickly store the state of a volume at a given time.
+Snapshots can be taken either manually or by utilizing the CSI driver.
+Any snapshot taken can be utilized to create new volumes.
+
+Currently only FSxZ volume level snapshots are supported. 
+FSxZ filesystem level backups are NOT supported.
+Snapshots can only be restored to volumes that exists under the same filesystem.
```

Comment:
I think we should mention that taking a volume snapshot of a PV with a ```volumeType``` of "filesystem" will create an FSxZ snapshot of the file system's root volume.